sentry/resource_sentry_project.go

```diff
@@ -248,30 +283,63 @@ func resourceSentryProjectUpdate(ctx context.Context, d *schema.ResourceData, me
 
 	d.SetId(proj.Slug)
 
+	oldTeams := map[string]bool{}
```

Comment:
use `map[string]struct{}{}` instead. It will use less memory :-)

Reply:
Thank you - fixed in https://github.com/Canva/terraform-provider-sentry/pull/33/commits/144799b773b30e9898efd2e41ad05a6ce1b299d2